Document/0x05g-Testing-Network-Communication.md

```diff
@@ -338,6 +338,7 @@ Dynamic analysis can be performed by launching a MITM attack with your preferred
 
 There are several ways to bypass certificate pinning for a black box test, depending on the frameworks available on the device:
 
+- Frida: Use the [Universal Android SSL Pinning Bypass with Frida](https://codeshare.frida.re/@pcipolloni/universal-android-ssl-pinning-bypass-with-frida/ "Universal Android SSL Pinning Bypass with Frida") script.
 - Objection: Use the `android sslpinning disable` command.
 - Xposed: Install the [TrustMeAlready](https://github.com/ViRb3/TrustMeAlready "TrustMeAlready") or [SSLUnpinning](https://github.com/ac-pm/SSLUnpinning_Xposed "SSLUnpinning") module.
 - Cydia Substrate: Install the [Android-SSL-TrustKiller](https://github.com/iSECPartners/Android-SSL-TrustKiller "Android-SSL-TrustKiller") package.
```

Comment:
Could you order the list according to the alphabet and make "Cydia.." the first item?

Reply:
Done.